src/BeamAdapter/component/controller/InterventionalRadiologyController.inl

```diff
@@ -161,7 +160,7 @@ void InterventionalRadiologyController<DataTypes>::init()
 
     Inherit::init();
 
-    reinit();
+    this->reinit();
```

Comment:
```suggestion
```
As reinit() was doing nothing before (even overriding the inherited class to force doing nothing actually), you can remove this line I think ?